### ISSUE TITLE
Select, Copy and Paste table data into Google Sheets / Excel

### ISSUE THREAD

**Issue by cesarpereira904** [trey-wallis/obsidian-notion-like-tables#452]
### Problem

Because sometimes users might need to grab data from the table, at the time being is not possible to select, copy and paste data.

### Solution

Allow to select the table data and paste it into Excel /  Google Sheets

**Comment by trey-wallis** [comment]
Related to #467

**Comment by trey-wallis** [comment]
@cesarpereira904 You can currently right click on a cell and the text will be copied to your clipboard

**Comment by MSBack** [comment]
> @cesarpereira904 You can currently right click on a cell and the text will be copied to your clipboard

I assume this FR is more in terms of copying several cells or the whole table at once by selecting (hold and drag) and copying.

**Comment by cesarpereira904** [comment]
Exactly, as mentioned in the previous comment,sometimes, at least in my case, I just need to copy paste the all table to paste on google sheets and share the table content with more people.

**Comment by trey-wallis** [comment]
Circling back to this. I really like this feature request. Will add it to the backlog

**Comment by 42ama** [comment]
@trey-wallis does this issue also means copy-paste feature, between different loom tables? Case it will be nice to have the opportunity to copy rows from one table to another.

**Comment by Grisly00** [comment]
I would also like this feature! Specifically, I would like to be able to copy and paste (or duplicate) a row in the same loom table.  I.e. if you have an entry that is 90% the same as the one above it is annoying to retype everything. Thanks!
